src/main.js

```diff
@@ -46,6 +46,23 @@ import {
 } from '@fortawesome/free-brands-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
 import ga from './ga';
+// Electron specific Code
```

Comment:
Is electron specific code is meant to be extended? Is it revelant to wrap it into a `util` module?

Reply:
I currently have no plan to extend it as the electron app solely needs to be able to write to the status screen other than that it handles everything else. But can move it to a util module if that is preferred

Reply:
i think it could be nice, therefore if needed it would not require any refactor. Don't you think?

Reply:
I will implement this change later when i fix the other two issues then for future eas incase i need to implement anything else

Reply:
I have left in the if statement in main to check and then if in electron it calls the init function from electron.js allowing code to be added to this in the future and no changes to be needed to main js for these